### ISSUE TITLE
User interface in multiple languages

### ISSUE THREAD

**Issue by iancoleman** [iancoleman/bip39#36]
It would be good to have the user interface work in multiple languages.

This is a two-phase issue:

1. Create logic for the user interface to display in different languages.
1. Create a corpus of all sentences in different languages to be used by the page.

**Comment by HostFat** [comment]
You can put the strings on https://www.transifex.com, it is free for open source projects.

**Comment by iancoleman** [comment]
Good suggestion.

I've started a project containing the phrases from the site - https://www.transifex.com/iancoleman/bip39-website/

**Comment by iancoleman** [comment]
Some progress:

* https://github.com/iancoleman/bip39/commit/60599f7ae9f59d6f0abadee1e0dd952ce4019d3d - Strings in UI marked for translation
* https://github.com/iancoleman/bip39/commit/c3c3df473e0e5114b75dbe835ada59a0e8066543 - Translation library does basic language switching

Also a request for translators at bitcointalk.org

https://bitcointalk.org/index.php?topic=1697804.0

**Comment by LeaTex** [comment]
I can help with spanish translation!

**Comment by iancoleman** [comment]
This never really gained any momentum.

The translation library was hand-rolled and rather than investigate replacement with an off-the-shelf library for a feature not actually in use, this feature has been removed (see https://github.com/iancoleman/bip39/commit/fb00a22d3fbce70c8694419a7378ece7dd93e283).

I've attached the Italian translations in case of future need, but have otherwise deleted the transifex project.
[for_use_bip39-website_enjson_it.txt](https://github.com/iancoleman/bip39/files/1347823/for_use_bip39-website_enjson_it.txt)

If this project should be desired in other languages, probably best to fork it and translate that way. If enough interest gathers around translations maybe it can be incorporated into the tool, but for now it's just unused cruft so is best to be removed.
